lib/Reader/readerir.cpp

```diff
@@ -3718,7 +3718,8 @@ IRNode *GenIR::callHelper(CorInfoHelpFunc HelperID, bool MayThrow, IRNode *Dst,
       (Dst == nullptr) ? Type::getVoidTy(LLVMContext) : Dst->getType();
   return (IRNode *)callHelperImpl(HelperID, MayThrow, ReturnType, Arg1, Arg2,
                                   Arg3, Arg4, Alignment, IsVolatile, NoCtor,
-                                  CanMoveUp).getInstruction();
+                                  CanMoveUp)
+      .getInstruction();
```

Comment:
I agree. We'll see how the formatting job feels about it, since it's likely that I'm running a slightly newer version of clang-format locally.